deployments/jwebbinar/image/env-frozen/jwebbinar/requirements.txt

```diff
@@ -1205,6 +1212,7 @@ wheel==0.37.0
     # via pip-tools
 widgetsnbextension==3.5.1
     # via
+    #   -r /opt/common-env/jupyter.pip
     #   -r /opt/environments/jwebbinar/jdaviz.pip
```

Comment:
This can be gotten rid of, the jdaviz kernel/env don't exist for jwebbinar anymore.  Pretty sure that was merged?

Reply:
Not sure what you mean.  Those lines are in the pip freeze/compile output for jwebbinar.   They indicate that widgetsnbextension is being pulled in by both jupyter.pip and jdaviz.pip.   jdaviz.pip is presumably what is adding jdaviz to the jwebbinar environment.   If it is listed explicitly in jdaviz.pip,  it should probably be removed from that since it is now in common-env/jupyter.pip.